packages/core/src/routes/admin-user/basics.ts

```diff
@@ -111,13 +116,29 @@ export default function adminUserBasicsRoutes<T extends AuthedRouter>(...args: R
         primaryEmail: string().regex(emailRegEx),
         username: string().regex(usernameRegEx),
         password: string().min(1),
+        passwordEncrypted: string(),
+        passwordEncryptionMethod: nativeEnum(UsersPasswordEncryptionMethod),
         name: string(),
       }).partial(),
       response: userProfileResponseGuard,
       status: [200, 404, 422],
     }),
     async (ctx, next) => {
-      const { primaryEmail, primaryPhone, username, password, name } = ctx.guard.body;
+      const {
+        primaryEmail,
+        primaryPhone,
+        username,
+        password,
+        name,
+        passwordEncrypted,
+        passwordEncryptionMethod,
+      } = ctx.guard.body;
+
+      assertThat(!(password && passwordEncrypted), new RequestError('user.password_and_encrypted'));
```

Comment:
Why should it have both password and passwordEncrypted. I thought having either one of them is OK, no?

Reply:
Either one of them is OK, but can not be exits both.

Reply:
Ah, I see. OK